assets/layers/indoors/indoors.json

```diff
@@ -474,6 +476,13 @@
           },
           "icon": "./assets/layers/indoors/room_office.svg"
         },
+        {
+          "if": "room=printer",
+          "then": {
+            "en": "This is a copy room"
+          },
+          "icon": "./assets/layers/indoors/room_printer.svg"
```

Comment:
Either we remove this line and go ahead, or 'room_printer.svg' is added with copyright information.

Reply:
Removed it for now.